keyboards/cantor/cantor.h

```diff
@@ -0,0 +1,35 @@
+// Copyright 2022 Diego Palacios (@diepala)
```

Comment:
This file should be removed as you have matrix data in `info.json`

Reply:
Removing this file produces errors

Reply:
Remove the layout macrco specifically, then

Reply:
It won't copmile:
```
☒ Matrix data out of bounds for layout LAYOUT_split_3x6_3 at index 6 (k4A): 4, 0
gmake[1]: *** [builddefs/build_keyboard.mk:335: .build/obj_cantor/src/layouts.h] Error 1
```